astrolabe/validator.py

```diff
@@ -73,19 +71,8 @@ def run_test(self, driver_workload):
 
         try:
             stats = subprocess.stop()
-        except TimeoutExpired:
-            self.fail("The workload executor did not terminate soon after "
-                      "receiving the termination signal.")
```

Comment:
Could you clarify why this was removed or if this logic was moved to another place?

Reply:
See: https://github.com/mongodb-labs/drivers-atlas-testing/pull/120/files#r613277944